File: Ohtap E Commerce Domain Models/Development/PHP/Objects/Invoices/Invoice.php
<?php
namespace Ohtap\ECommerce\DomainModels\Objects\Invoices;

class Invoice  implements IInvoice
{

    public $Photo;//FileBasic
    public $Customer;//CustomerBasic
    public $Date;
    public $TotalPrice;
    public $ID;
    public $Title;
    public $State;
    public $OrderNo;
    public $Description;
}

File: Ohtap E Commerce Domain Models/Development/NetFramework/Models Objects/Models Objects/Invoices/Invoice.cs
using Ohtap.ECommerce.Domain.Models.Objects.Products;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Ohtap.ECommerce.Domain.Models.Objects.Invoices
{
    public class Invoice : InvoiceBasic, IInvoice
    {
        public virtual Customers.ICustomerData Customer { get; set; }
        public virtual Files.IFileData Photo { get; set; }
    }
}
